packages/edge-login-ui-rn/src/constants/Scenes.ts

```diff
@@ -0,0 +1,27 @@
+export interface Scene {
+  title: string
+  subTitle: string
+}
+
+export enum SceneNames {
```

Comment:
While it's not "deprecated" per-se, nobody really likes the TypeScript `enum` type, because it's not part of the ECMA-262 JavaScript standard. Generally speaking, TypeScript is pretty much "plain JavaScript plus types", so you can just strip the types to get valid JS. The `enum` keyword, on the other hand, actually requires code-generation, since it's a new language feature that exists at run-time.

Instead of using `enum`, the modern thing is to just use `type SomeEnum = 'a' | 'b' | 'c'`, which gives the same benefits while being a pure compile-time construct. This is because, if you have `let x: SomeEnum`, you can safely write code like `x = 'a'`. If you make a typo like `x = 'd'` or `if (x === 'd')`, TypeScript will complain because the types say that can't happen. So, the raw strings are actually safe.

Now, I realize that in most other languages raw strings are incredibly dangerous. Things like `x = 'a'` are a huge red flag, because there is no type-checking. Therefore, the safe move in C / C++ / C# etc. is to define some variables with the constants, and use those instead. However, this conventional wisdom doesn't apply to TypeScript, where we *do* have type checking on the raw strings! Plus, the extra constants just add size & extra work for the interpreter (unlike a compiled language where they might increase performance).

In other words, the safest & most efficient thing to do is to drop the `enum` keyword and just use raw strings. So, please remove `SceneNames` and just use the strings. It's OK, I promise.